build.gradle

```diff
@@ -92,20 +86,17 @@ allprojects {
     testCompile group: 'junit', name: 'junit', version: '4.13'
   }
 
-  dependencyUpdates.resolutionStrategy {
-  componentSelection { rules ->
-    rules.all {
-      def isNonStable = { String version ->
-        ['alpha', 'beta', 'rc', 'cr', 'm', 'preview', 'b', 'ea'].any { qualifier ->
-          version ==~ /(?i).*[.-]\$qualifier[.\\d-+]*/
-        }
-      }
-      if (isNonStable(candidate.version) && !isNonStable(currentVersion)) {
-        selection.reject('Release candidate')
-      }
+  // before committing a change, make sure task still works
+  dependencyUpdates {
```

Comment:
this seems to be duplicated in the file?

Reply:
back in a day, i think used to not work if only inside `allprojects` section

Reply:
worked when removed :shrug: